### DISCUSSION TITLE
ルールに関する疑問点・改善点

### DISCUSSION THREAD

**Discussion by kerochan** [scp-tcg-jp-j/rule#5] [Q&A]
ルールに関する疑問点・改善点の提案はここにコメントしてください

**Comment by smctrl** [comment]
@kerochan さん
すみません、本来プルリク作るべきでしたが https://github.com/scp-tcg-jp-j/rule/blob/develop/README.md を直接pushしてしまいました！  
# 改善点
「releaseブランチ」「developブランチ」といったGitの専門用語を知らなくても策定済みルールに飛べるようにリンクを作成しました。
